simple_make.sh is broken

simple_make.sh is broken
OS:
`uname -a`
Linux client0 4.4.0-53-generic #74-Ubuntu SMP Fri Dec 2 15:59:10 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
`lsb_release -a`
No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 18 Sarah
Release:	18
Codename:	sarah

qTox version: v1.8.1-135-gf72c83a.
Commit hash:  f72c83afb76e4c4f055cb3e5bad9447bd03475e1

Relevant log snippit:

make[1]: Leaving directory '/home/avril/Development/qTox/libs/libtoxcore-v0.1.2'
~/Development/qTox
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for one of the modules 'libavcodec'
-- Checking for one of the modules 'libavdevice'
-- Checking for one of the modules 'libavformat'
-- Checking for one of the modules 'libavutil'
-- Checking for one of the modules 'libqrencode'
-- Checking for one of the modules 'libsodium'
-- Checking for one of the modules 'libswscale'
-- Checking for one of the modules 'sqlcipher'
-- Checking for one of the modules 'vpx'
-- Checking for one of the modules 'toxcore'
-- Checking for one of the modules 'toxav'
-- Checking for one of the modules 'toxencryptsave'
-- Checking for one of the modules 'libtoxcore'
-- Checking for one of the modules 'libtoxav'
-- Checking for one of the modules 'openal'
-- Checking for one of the modules 'x11'
-- Checking for one of the modules 'gdk-pixbuf-2.0'
-- Checking for one of the modules 'glib-2.0'
-- Checking for one of the modules 'gtk+-2.0'
-- Checking for one of the modules 'check'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/avril/Development/qTox/_build
make: *** No targets specified and no makefile found.  Stop.

I was able to finish compiling manually by CDing to _build and entering `make`, so it's probably a very simple working directory problem. I reproduced it after cloning the repo again, so I'm pretty sure about this one.

--- a/simple_make.sh
+++ b/simple_make.sh
@@ -119,8 +119,9 @@ zypper_install() {
     local zypper_packages=(
         automake
         cmake
-        ffmpeg2-devel
         git
+        libavcodec-devel
+        libavdevice-devel
         libopus-devel
         libQt5Concurrent-devel
         libqt5-linguist

--- a/simple_make.sh
+++ b/simple_make.sh
@@ -117,51 +117,59 @@ fedora_locallib() {
 
 zypper_install() {
     local zypper_packages=(
+        automake
         cmake
+        ffmpeg2-devel
         git
+        libopus-devel
         libQt5Concurrent-devel
+        libqt5-linguist
+        libqt5-linguist-devel
         libQt5Network-devel
         libQt5OpenGL-devel
-        libQt5Sql-devel
-        libQt5Sql5-sqlite
-        libQt5Xml-devel
-        libXScrnSaver-devel
-        libffmpeg-devel
-        libopus-devel
-        libqt5-linguist
         libqt5-qtbase-common-devel
         libqt5-qtsvg-devel
+        libQt5Sql5-sqlite
+        libQt5Sql-devel
+        libQt5Test-devel
+        libQt5Xml-devel
         libsodium-devel
         libvpx-devel
+        libXScrnSaver-devel
         openal-soft-devel
-        patterns-openSUSE-devel_basis
         patterns-openSUSE-devel_basis
         qrencode-devel
         sqlcipher-devel
     )
+
+    # if not sudo is installed, e.g. in docker image, install it
+    command -v sudo || zypper in sudo
+
     sudo zypper in "${zypper_packages[@]}"
 }
 
 main() {
-    if which apt-get
+    if command -v zypper && [ -f /etc/products.d/openSUSE.prod ]
+    then
+        zypper_install
+    elif command -v apt-get
     then
         apt_install
-    elif which pacman
+    elif command -v pacman
     then
         pacman_install
-    elif which dnf
+    elif command -v dnf
     then
         dnf_install
         fedora_locallib
-    elif which zypper
-    then
-        zypper_install
     else
         echo "Unknown package manager, attempting to compile anyways"
     fi
 
     ./bootstrap.sh
-    cmake -H. -B_build
+    mkdir -p _build
+    cd _build
+    cmake ../
     make -j$(nproc)
 }
 main

